src/components/App/App.js

```diff
@@ -28,17 +31,34 @@ class App extends Component {
 					Please take an item and show it to the camera
 				</header>
 				<hr />
-				<WebcamCaptureContainer confirmMatch={this.confirmMatch} />
-				{this.state.prediction && (
+				Slack Username:
+				<input
+					value={this.props.currentUser}
+					onChange={event =>
+						this.changeCurrentUser(event.target.value)
+					}
+				/>
+				<hr />
+				<WebcamCaptureContainer confirmMatch={this.setPrediction} />
+				{this.props.prediction && (
 					<ConfirmationBoxContainer
-						item={this.state.prediction.index}
-						onYes={() => this.setState({prediction: null})}
-						onNo={() => this.setState({showList: true})}>
-						<img src={this.state.prediction.img} alt="" />
+						item={this.props.prediction.index}
+						onYes={() => this.props.setPrediction(null)}
+						onNo={() => {
+							this.props.setPrediction(null);
+							this.props.setShowList(true);
+						}}>
+						<img src={this.props.prediction.img} alt="" />
 					</ConfirmationBoxContainer>
 				)}
-				{this.state.showList && (
-					<StoreList username={this.state.currentUser} />
+				{this.props.showList && (
+					<StoreListContainer
+						storeList={this.props.storeList}
+						loadStoreListError={this.props.loadStoreListError}
```

Comment:
Do either of these props need to be passed down like this? App doesnt seem to need them